projects/common/src/lib/utils/pipes/data.pipes.ts

```diff
@@ -238,7 +238,23 @@ export class DataPipes implements PipeTransform {
      */
 
     if(format === DataPipeConstants.PIPE_STRING_SLICE_FIFTY.toLowerCase()) {
-      return val.slice(0, 50);
+      return val.slice(0, 50) + "...";
```

Comment:
I assume you're getting lint errors on if() and using double quotes (""), need to try and prevent this. Not a big deal, just going forward.